interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -89,6 +91,10 @@ object TapirAdapter {
       oneOfVariantValueMatcher[CalibanBody.Stream[stream.BinaryStream]](
         streamTextBody(stream)(CodecFormat.Json(), Some(StandardCharsets.UTF_8)).toEndpointIO
           .map(Right(_)) { case Right(value) => value }
+      ) { case Right(_) => true },
+      oneOfVariantValueMatcher[CalibanBody.Stream[stream.BinaryStream]](
+        streamBinaryBody(stream)(CodecFormat.TextEventStream()).toEndpointIO
+          .map(Right(_)) { case Right(value) => value }
```

Comment:
I'm quite surprised this works since there is a `Right(_) => true` above this, does the selector take into account the Accepts headers when computing the match?